src/py/pyodide/http.py

```diff
@@ -138,9 +144,11 @@ def url(self) -> str:
         """
         return self.js_response.url
 
-    def _raise_if_failed(self) -> None:
+    def _raise_if_failed_or_aborted(self) -> None:
```

Comment:
Maybe just leave the name alone? I feel like it's clear enough and with `_or_aborted` this is a bit of a mouthful.

Reply:
Thanks. I am really bad at naming things😂